Resources/Prototypes/SimpleStation14/Entities/Clothing/OuterClothing/hardsuits.yml

```
@@ -27,3 +27,31 @@
   - type: ToggleableClothing
     clothingPrototype: ClothingHeadHelmetHardsuitDominia
 
+- type: entity
+  parent: ClothingOuterHardsuitBase
+  id: ClothingOuterHardsuitHoP
+  name: head of personnel's plated hardsuit
+  description: A stylish hardsuit for only the most esteemed. Doesn't offer much damage protection.
+  components:
+  - type: Sprite
+    sprite: SimpleStation14/Clothing/OuterClothing/Hardsuits/hopsuit.rsi
+  - type: Clothing
+    sprite: SimpleStation14/Clothing/OuterClothing/Hardsuits/hopsuit.rsi
+  - type: PressureProtection
+    highPressureMultiplier: 0.02
+    lowPressureMultiplier: 10000
```

```suggestion
    lowPressureMultiplier: 4500
```

5500 Seems to be about standard for non-combat hardsuits, so I'd think this would make sense for HoP.